frontend/src/features/admin/edit-user/EditUserPage.tsx

```diff
@@ -104,7 +112,7 @@ const EditUserPage = (props: IEditUserPageProps) => {
     note: user.note,
     agency: user.agencies && user.agencies.length !== 0 ? user.agencies[0].id : '',
     role: user.roles && user.roles.length !== 0 ? user.roles[0].id : '',
-    position: user.position,
+    position: user.position ?? '',
```

Comment:
had to initialize the position property to an empty string here if the user.position is null, otherwise the browser would give the warning: A component is changing an uncontrolled input to be controlled.